source/partials/_nav.slim

```diff
@@ -5,20 +5,21 @@
   = nav_overlay_item('Apprenticeship', '/apprenticeship')
   = nav_overlay_item('Blog', '/blog')
 
-nav#main-nav.Nav.Nav--topPositioned class="#{current_page.data.nav_class}" aria-label=('Main navigation')
-  = link_to '/', class: 'Nav-logo'
-    .NavLogo class=nav_logo_class
+header
+  #main-nav.Nav.Nav--topPositioned class="#{current_page.data.nav_class}"
```

Comment:
I just followed what is being done in the `footer`, thought it was some SUIT thing. :p